src/puzzle-time/pm-puzzle-time.ts

```diff
@@ -0,0 +1,41 @@
+import "./puzzle-time.css";
```

Comment:
This works and I like it.  I think this is a good start and could be changed to persist the time a player has spent on a puzzle in future iterations.